code/game/objects/items/weapons/implants/implant_mindshield.dm

```diff
@@ -12,31 +12,22 @@
 				<HR>
 				<b>Implant Details:</b><BR>
 				<b>Function:</b> Contains a small pod of nanobots that manipulate the host's mental functions.<BR>
-				<b>Special Features:</b> Will prevent and cure most forms of brainwashing.<BR>
+				<b>Special Features:</b> Will prevent most forms of brainwashing.<BR>
 				<b>Integrity:</b> Implant will last so long as the nanobots are inside the bloodstream."}
 	return dat
 
 
 /obj/item/implant/mindshield/implant(mob/target)
 	if(..())
-		if(target.mind in SSticker.mode.head_revolutionaries || is_shadow_or_thrall(target))
```

Comment:
Meh on this. Shadowlings are easy to identify (they're dying because of the flashlight in your pocket). Thralls are also easy to identify (remove their mask, examine them). So... I'm not sure this matters much either way.